src/actions/guides/getting-started/installing.cr

```diff
@@ -59,17 +59,11 @@ class Guides::GettingStarted::Installing < GuideAction
     ### On macOS
 
     * Install [Homebrew](https://brew.sh/)
-    * Run `brew install openssl` to make sure you have OpenSSL
     * Run `brew tap luckyframework/homebrew-lucky`
     * Run `brew install lucky`
     * Make sure [Postgres CLI tools](https://postgresapp.com/documentation/cli-tools.html)
       are installed if you're using Postgres.app
-    * If you are on macOS High Sierra you need to add `export PKG_CONFIG_PATH=/usr/local/opt/openssl/lib/pkgconfig`
-    to your `~/.bash_profile` or `~/.zshrc`
 
-    > If you get an error like this: `Package libssl/libcrypto was not found in the
```

Comment:
I’m not sure this actually should be removed. I tried installing and still needed it so I dunno. Can you give it a try?

Reply:
Did you remove your PKG_CONFIG and then tried installing and got errors? I thought that's what https://github.com/Homebrew/homebrew-core/pull/48028 was fixing, but maybe that's not actually fixed? I don't mind adding it back in, I just thought it was all fixed 🤷‍♂  Let me know what steps you tried, and I'll try the same thing.